extensions/2.0/Vendor/EXT_mesh_features/schema/propertyTable.property.schema.json

```diff
@@ -68,18 +68,18 @@
                 }
             ]
         },
-        "stringOffsetBufferView": {
+        "stringOffsets": {
```

Comment:
I noticed that this spec doesn't explicitly mention `stringOffsets`. When describing types, it would be good to mention that `stringOffsets`/`arrayOffsets` are required in some cases. It doesn't have to be very long of an explanation, just link to https://github.com/CesiumGS/3d-tiles/tree/main/specification/Metadata#strings or https://github.com/CesiumGS/3d-tiles/tree/main/specification/Metadata#arrays

Reply:
Acknowledged – currently this PR is a draft meant to help reach a decision on accessor vs. buffer view storage, whereas that change would be wanted regardless. Probably best as a separate PR?